gaj23/face-it
# Gaj/documentation/update 03 02

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

This finishing error and loading testing.

This holds updates as of 02/03 for the README

### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.

A problem with interception was found during the error testing process. I discovered that further mocking of user flow was necessary that forced errors to occur.


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Please see the cypress folder. 

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Please insure that you have cypress downloaded and running in your terminal. 
Please run all tests. Please note that due to fetching, sometimes there's a delay in the data loading and multiple refreshing might be necessary before the data is fully loaded. 


### Steps To Take
> Any questions and possible steps moving forward.

Are there any further sad path testing that we should consider? 


#### Checklist:
